### ISSUE TITLE
RFC 9266: Channel Bindings for TLS 1.3 support

### ISSUE THREAD

**Issue by Neustradamus** [UWPX/UWPX-Client#178]
Can you add the support of RFC 9266: Channel Bindings for TLS 1.3?
- https://datatracker.ietf.org/doc/html/rfc9266

Little details, to know easily:
- tls-unique for TLS =< 1.2
- tls-exporter for TLS = 1.3

Thanks in advance.

Linked to:
- https://github.com/UWPX/UWPX-Client/issues/51

**Comment by COM8** [comment]
Yes, I would like to, but there is currently no support for TLS1.3 build into C# (UWP). I was planing to enable TLS1.3 a couple of months ago and there weren't any plans from MS to add support for it, since UWP is now a "deprecated" platform.

**Comment by Neustradamus** [comment]
@COM8: Where is the code exactly?
We can create a ticket to add this feature...

**Comment by Neustradamus** [comment]
@COM8: Have you progressed?

Since jabber.ru MITM, a lot of devs wake up ^^

**Comment by COM8** [comment]
https://github.com/UWPX/UWPX-Client/releases/tag/v.0.43.0.0
